dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -72,7 +72,7 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
     vMethods.toArray.map{case m: String => m.toLowerCase()}.map {
       case "top1" => new Top1Accuracy[T]()
       case "top5" => new Top5Accuracy[T]()
-      case "loss" => new Loss[T]()
+      case "loss" => new Loss[T](new ClassNLLCriterion[T]())
```

Comment:
We should allow the python user to specify what criterion to use for testing "loss". @zhichao-li